modules/apps/frontend-theme/frontend-theme-classic/src/css/custom_properties/_custom_properties_variables.scss

```diff
@@ -1,7 +1,7 @@
 // Spacers map copy
 
-$spacers-cp: ();
-$spacers-cp: map-deep-merge(
+$custom-properties-spacers: ();
+$custom-properties-spacers: map-deep-merge(
```

Comment:
We don't need `map-deep-merge` here. There is only one level of maps, `map-merge` is good enough. We only need `map-deep-merge` if we have a map nested inside the map.